packages/xrpl/src/models/methods/ammInfo.ts

```diff
@@ -0,0 +1,81 @@
+import { Amount, IssuedCurrencyAmount } from '../common'
+
+import { BaseRequest, BaseResponse } from './baseMethod'
+
+/**
+ * The `amm_info` command retrieves information about an AMM instance.
+ * Returns an {@link AMMInfoResponse}.
+ *
+ * @category Requests
+ */
+export interface AMMInfoRequest extends BaseRequest {
+  command: 'amm_info'
+
+  /**
+   * A hash that uniquely identifies the AMM instance.
+   */
+  amm_id?: string
+
+  /**
+   * Specifies one of the pool assets (XRP or token) of the AMM instance.
+   */
+  asset1?: Amount
+
+  /**
+   * Specifies the other pool asset of the AMM instance.
+   */
+  asset2?: Amount
+}
```

Comment:
I assume that these two have to be specified together - I'd recommend including that requirement in their docstrings. (Ex. "Specifies one of the pool assets (XRP or token) of the AMM instance. Both asset1 and asset2 must be defined to specify an AMM instance.")